### ISSUE TITLE
Giving the ability to manage secrets with SealedSecret in an umbrella chart

### ISSUE THREAD

**Issue by idrissneumann** [AdWerx/charts#22]
The secrets `xxxx-secret-key` are requiring a `secret_key` in the value.yaml.

We'd like to handle the creation of those secrets using a SealedSecret template like that in our umbrella chart:

```yaml
{{- range $key := index .Values "sealed-secrets" }}
---
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  name: {{ .name }}
spec:
  encryptedData:
    {{- .data | toYaml | nindent 4 }}
  template:
    metadata:
      labels:
        {{- if .labels }}
        {{- .labels | toYaml | nindent 8 }}
        {{- end}}
      name: {{ .name }}
    type: {{ .type | default "Opaque" | quote }}
{{- end }}
```

This way, we'll be able to avoid to push clear password values in our gitops repositories. So in order to achieve that, we'll need to make the `secret_key` attribute as optional. If it's not defined, this template will not create a secret: https://github.com/AdWerx/charts/blob/master/awx/templates/secret-key-secret.yaml

Basically we just have to add a `if` in this template.

Thanks in advance.

**Comment by idrissneumann** [comment]
I opened this pull request in order to fix this problem we have: https://github.com/AdWerx/charts/pull/23/files

Thanks

**Comment by jbielick** [comment]
Thanks for the info, @idrissneumann 

I think this is a great idea. To capture a lot of use-cases and follow some Helm patterns in other charts, I've made some changes to the chart to allow an "Existing Secret" to be used in several places. I think that would resolve your use case here. Let me know if that's not correct.

For version 3.1.0, you could provide:

```
secretKeyExistingSecret: name-of-sealed-secret
```

and inside the `name-of-sealed-secret` secret, you'll want a 

```
  SECRET_KEY: ...
```

key and value. The values description table has more info.

Let me know how that works for you.

**Comment by idrissneumann** [comment]
I think it'll do the trick, thanks :)
